concepts.wip/if-expressions/about.md

```diff
@@ -1,52 +1,87 @@
 # About
 
-`if`-expressions in Julia are similar to those seen in other languages:
+In Julia, like many other languages, allow for the following with `if`-expressions:
+
+- Chained if-expressions
+- Nested if-expressions
+
+~~~~exercism/note
+[`if`-expressions](https://docs.julialang.org/en/v1/manual/control-flow/#man-conditional-evaluation) as defined here will encompass the following syntaxes:`if`, `if-else` and `if-elseif-else` statements.
+~~~~
+
+`if`-expressions allows for branching in our programs, given a condition is satisfied.
+
+## A first example
+
+The `function` defined below is not very realistic, but it shows the anatomy of an `if` statement. 
+
+```julia
+julia> function password_checker(password)
+           if password == system_password
+               return "Welcome back!"
+            end
+        end
+
+# Assume system_password is "abc123"
+julia> password_checker("abc123")
+Welcome back!
+
+julia> password_checker("xyz321")
+
+```
+`println("Welcome back!")` is run, if the `password == system_password`, otherwise it is skipped, and nothing is printed to the screen.
+
+We can do the following for effective branching:
+
+## Chained expressions example
 
 ```julia
-julia> function say_if_positive(n)
-           if n > 0
-               println("n is positive!")
-           else
-               println("n is not positive!")
-           end
-       end
-say_if_positive (generic function with 1 method)
-
-julia> say_if_positive(10)
-n is positive!
-
-julia> say_if_positive(-10)
-n is not positive!
+# We can test for multiple cases
+# with chained if expression.
+if 0 ≤ n ≤ 0.5
+    println("n is in the interval [0, 0.5], inclusive.")
+elseif -0.5 ≤ n < 0
+    println("n is in the interval [-0.5, 0).")
+elseif n > 0.5
+    println("n is greater than 0.5.")
+else
+    println("n is less than -0.5.")
+end
 ```
 
-<!-- TODO: Add that fancy concept highlight embed thing to boolean expression -->
+In the above chained expression, we can have mulitple `elseif` statements, with only one `else` statement at the end, but it's not necessary. 
+
+## Nested expressions example
 
-If the boolean expression following the `if` evaluates to `true`, the first block of code is run and the second block is skipped.
-If the boolean expression following the `if` evaluates to `false`, the first block of code is skipped and the second block is run.
-The program continues running at the first line of code after the `end` keyword.
+```julia
+# We can refine our coniditonal evaluation
```

Comment:
Probably better to have the comments in the markdown rather than as code comments?